src/content/docs/en/guides/astro-db.mdx

```diff
@@ -130,11 +135,13 @@ const Comment = defineTable({
 });
 ```
 
-## Seed your database
+## Seed your database for development
 
-In development, Astro will use your DB config to generate local types according to your schemas. These will be generated fresh each time the dev server is started, and will allow you to query and work with the shape of your data with type safety and autocompletion.
+In development, Astro will use your DB config to generate local types according to your schemas. These will be generated fresh from your seed file each time the dev server is started, and will allow you to query and work with the shape of your data with type safety and autocompletion.
 
-To seed development data for testing and debugging into your Astro project, create a `db/seed.ts` file. Import both the `db` object and any configured table from `astro:db`. Use the `db.insert()` function to provide an array of table row data objects.
+You will not have access to production data during development unless you [seed from your remote database](#seed-from-your-remote-database). This protects your data while allowing you to test and develop with type-safety using simulated data.
```

Comment:
Maybe the link can be removed. It seems that it is not recommended for the most novice since it involves a risk. More advanced users will know how to do it with the rest of the doc I think... But maybe I'm wrong. 😅 
```suggestion
You will not have access to production data during development unless you seed from your remote database. This protects your data while allowing you to test and develop with type-safety using simulated data.
```

Reply:
You are correct; this seems to be the appropriate course of action. Perhaps we should wait for the others so we can consider everyone's input.

Reply:
**If** a similar section exists, just with a different title, then I would suggest updating the link to that new section slug. My reasoning is:

 "You won't be able to have X unless you do Y" leads to, "OK, yes, I want to do Y! How do I do that!" This is exactly where a link is helpful.

If you have restructured so that there is no comparable section, it's fine to remove the link. BUT, don't just remove the link to "fix links" because *I* can go in and address issues that are problems in e.g. other files. That's not a blocker here. Decide whether to leave a link in based on whether it is *helpful to the reader*!